pkg/output/progress.go

```diff
@@ -98,3 +121,19 @@ Loop:
 	}
 	logrus.Debug("Progress did not receive any tic. Stopping...")
 }
+
+func (p *progress) flush() {
+	for i := 0; i < p.highestLineLength; i++ {
+		Printf(" ")
+	}
+	Printf("\r")
+}
+
+func (p *progress) printf(format string, args ...interface{}) {
+	length := len(fmt.Sprintf(format, args...))
```

Comment:
why don't you save the sprintf result to print it at the end instead of doing it 2 times ?